src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
```

Comment:
this adds us no benefits. We need to throw custom exceptions or more explicit ones, if they exists. 
Also it will be better to pass meaningful message to our exception, so it will tell us something. 
Like `throw new IllegalStateException("com.mysql.cj.jdbc.Driver not found")`